src/widgets/mapping.js

```diff
@@ -11,7 +11,6 @@ export default {
 	'SchemaOneOf': require('./oneOf.vue').default,
 	'SchemaEnum': require('./Enum.vue').default,
 	'widget-googlemaps': require('./widget-googlemaps.vue').default,
```

Comment:
do we use this component anywhere?
widget-googlemaps

Reply:
It doesn't seem so, let's remove that one too